onebusaway-android/src/main/java/org/onebusaway/android/provider/ObaContract.java

```diff
@@ -646,6 +635,33 @@ public static boolean markAsUnused(Context context, Uri uri) {
             values.putNull(ObaContract.Stops.ACCESS_TIME);
             return cr.update(uri, values, null, null) > 0;
         }
+
+        public static int getArrivalFilter(Context context, Uri uri) {
```

Comment:
What do you think about changing the return type here to the enum `ArrivalFilter`?  Seems like a conversion always needs to be done anyway, so we might as well do it here.